packages/flame/svg/Icons/icon-slice.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><g id="slice" fill="#232A3B"><path id="base-1" d="M15.94 6.082a7.004 7.004 0 0 0-5.929-6.01A.885.885 0 0 0 9 .949V6a1 1 0 0 0 1 1h5.135a.812.812 0 0 0 .805-.918z"/><path id="base-2" d="M6.025 2.067A7.001 7.001 0 0 0 7 16c3.48 0 6.367-2.54 6.909-5.867A.976.976 0 0 0 12.945 9H8a1 1 0 0 1-1-1V2.916a.857.857 0 0 0-.975-.849z" opacity=".7"/></g></svg>
```

Comment:
ran `yarn svgo` script, this icon is not from the ones I'm adding and has a default color instead of being b/w, should I keep as-is or revert this?
With fill:
![image](https://user-images.githubusercontent.com/19352322/80841356-34e1ef00-8bcd-11ea-854d-16bfa9429f32.png)
Without fill:
![image](https://user-images.githubusercontent.com/19352322/80841382-45926500-8bcd-11ea-91b4-a4643b4ec88e.png)

Reply:
Should probably be without fill to keep in line with other icons that are shades of grey.
The one with fill has a bit of a blueish tint, which wouldn't match other icons